dbt_subprojects/hourly_spellbook/models/_metrics/fees/metrics_fees_index_daily.sql

```diff
@@ -0,0 +1,57 @@
+{{ config(
+        schema = 'metrics'
+        , alias = 'fees_index'
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['blockchain', 'block_date']
+        , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+        )
+}}
+
+{% set baseline_date = '2018-01-01' %}
+
+with baseline as (
+    select
+        sum(gas_fees_usd) as baseline_gas_fees_usd -- sum is required due to blockchain being second unique key in source
+    from
+        {{ ref('metrics_gas_fees_daily') }}
+    where
+        block_date = date '{{ baseline_date }}'
+), blockchain_baseline as (
+    select
+        blockchain
+        , gas_fees_usd as blockchain_baseline_gas_fees_usd
+    from
+        {{ ref('metrics_gas_fees_daily') }}
+    where
+        block_date = date '{{ baseline_date }}'
+), daily as (
+    select
+        blockchain
+        , block_date
+        , gas_fees_usd
+    from
+        {{ ref('metrics_gas_fees_daily') }}
+    where
+        block_date >= date '{{ baseline_date }}'
+        {% if is_incremental() %}
+        and {{ incremental_predicate('block_date') }}
+        {% endif %}
+)
+select
+    d.blockchain
+    , d.block_date
+    , d.gas_fees_usd
+    , b.baseline_gas_fees_usd
+    , bb.blockchain_baseline_gas_fees_usd
+    , (d.gas_fees_usd / b.baseline_gas_fees_usd) * 100 as fees_index
+    , (d.gas_fees_usd / bb.blockchain_baseline_gas_fees_usd) * 100 as blockchain_fees_index
```

Comment:
we get a lot of nulls here since chains don't find a match to date

Reply:
(note: applies to all three fees/txs/transfers spells)

Reply:
Sorry, I forgot to remove this from the prototype. More details [here](https://github.com/duneanalytics/spellbook/pull/6924/files#r1796743707), but tl;dr you can delete this from all the index calcs